gno.land/cmd/gnoland/testdata/maketx_call_sponsor.txtar

```diff
@@ -0,0 +1,26 @@
+# test for maketx call with sponsorship
+loadpkg gno.land/r/foo/bar $WORK
+
+# start a new node
+gnoland start
+
+## execute GetCaller
+gnokey maketx call -pkgpath gno.land/r/foo/bar -func GetCaller -gas-fee 1000000ugnot -gas-wanted 2000000 --sponsor=${USER_ADDR_test1} -broadcast -chainid=tendermint_test test1
```

Comment:
this command sponsors itself, it should be test1 sponsoring test2

Reply:
removed